CHANGELOG.md

```diff
@@ -7,3 +7,17 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## Added
 - Basic Skel
 - missing PR template
+
+## [0.0.2] - 2016-08-11
```

Comment:
Understood. Will it matter that the version is listed as 0.0.2 on rubygems or will this lead to confusion?

https://rubygems.org/gems/sensu-plugins-nrpe